central/signatureintegration/datastore/validate.go

```diff
@@ -28,27 +30,24 @@ func ValidateSignatureIntegration(integration *storage.SignatureIntegration) err
 		err := errors.New("name field must be set")
 		multiErr = multierror.Append(multiErr, err)
 	}
-	if integration.GetCosign() == nil {
-		err := errors.New("integration must have at least one signature verification config")
-		multiErr = multierror.Append(multiErr, err)
+	if len(integration.GetCosign().GetPublicKeys()) == 0 && len(integration.GetCosignCertificates()) == 0 {
+		multiErr = multierror.Append(multiErr, errors.New("integration must have at least one signature verification config"))
```

Comment:
I'm not sure if the error message is right. We check if there are no public keys and no certs should we give use a hint about it?